generator/explores/metric_definitions_explore.py

```diff
@@ -42,21 +45,22 @@ def from_dict(name: str, defn: dict, views_path: Path) -> MetricDefinitionsExplo
 
     def _to_lookml(
         self,
-        bq_client: bigquery.Client,
-        v1_name: Optional[str],
+        _bq_client: bigquery.Client,
+        _v1_name: Optional[str],
     ) -> List[Dict[str, Any]]:
-        intervals = ["date", "week", "month", "quarter", "year", "raw"]
+        exposed_fields = ["ALL_FIELDS*"]
+        if self.views["base_view"] != "client_counts":
```

Comment:
Ok I'm a bit confused what is happening here now ...

looking at looker it seems that you are using the `client_counts` view as the base table? which is not part of the whole metrics_definitions_* views

however ... looking at that in looker: [`client_counts`](https://mozilla.cloud.looker.com/projects/looker-hub/files/fenix/views/client_counts.view.lkml) extends [`baseline_clients_daily_table`](https://mozilla.cloud.looker.com/projects/looker-hub/files/fenix/views/baseline_clients_daily_table.view.lkml) which currently has the `client_id` dimension hidden (line 38) 
So currently nowhere in the explore we are able to select a client_id

Reply:
and client_counts has sooo many many fields that I'm not sure I like it for the base fields to be honest ... but that is more of a taste question

Reply:
Seeing `client_id` isn't super useful in most cases, so having it hidden should be fine.
`client_counts` does have a bunch of dimensions, but a lot of them can be quite useful.

I imagine the explores will be used to answer questions like: What was the total number of active hours (which is a metric coming from metric hub) for all clients on release over a specific time period? What was the average search count (also a metric-hub metric) per country for clients that have FF as default browser?

Reply:
ah ok that does make sense .. I was just primed to look for that now :)